contracts/UniswapV2Migrator.sol

```diff
@@ -24,25 +23,28 @@ contract UniswapV2Migrator is IUniswapV2Migrator {
     }
 
     function _safeTransferETH(address to, uint value) private {
-        (bool success,) = to.call.value(value)(new bytes(0));
+        (bool success,) = to.call{value: value}(new bytes(0));
         require(success, 'ETH_TRANSFER_FAILED');
     }
 
-    constructor(address _factoryV1) public {
+    constructor(address _factoryV1, address _router) public {
         factoryV1 = IUniswapV1Factory(_factoryV1);
+        router = IUniswapV2Router01(_router);
     }
 
     // needs to accept ETH from any v1 exchange and the router. ideally this could be enforced, as in the router,
     // but it's not possible because it requires a call to the v1 factory, which takes too much gas
-    function() external payable {}
+    receive() external payable {}
 
-    function migrate(address token, uint amountTokenMin, uint amountETHMin, address to, uint deadline) external {
+    function migrate(address token, uint amountTokenMin, uint amountETHMin, address to, uint deadline)
+        external override
```

Comment:
Extraneous newline?  This doesn't seem to match the style elsewhere, though the style in this file for function definitions is generally inconsistent, but this one seems to be an outlier even considering that.

Reply:
it's because the line exceeds the 120-character line length limit. you're right that it's internally inconsistent though, `external` and `override` should each have their own line. good catch.